drivers/sensor/lis2dh/lis2dh.c

```diff
@@ -114,7 +113,7 @@ static int lis2dh_sample_fetch(const struct device *dev,
 	return -ENODATA;
 }
 
-#ifdef CONFIG_LIS2DH_ODR_RUNTIME
+#if defined(CONFIG_LIS2DH_ODR_RUNTIME)
```

Comment:
Again why did you change it ? The reason I am asking is the less changes you have the less comments you will get from the community

Reply:
Mostly code consistency.